test/parallel/test-crypto-random.js

```diff
@@ -339,6 +339,14 @@ assert.throws(
   assert.ok(desc);
   assert.strictEqual(desc.configurable, true);
   assert.strictEqual(desc.enumerable, false);
+
+  const newVal = Symbol();
+  crypto[f] = newVal;
+
+  {
+    const desc = Object.getOwnPropertyDescriptor(crypto, f);
+    assert.strictEqual(desc.value, newVal);
```

Comment:
The most important thing to test imo is that the updated property is now enumerable.
```suggestion
    assert.strictEqual(desc.value, newVal);
    assert.strictEqual(desc.enumerable, true);
```